### ISSUE TITLE
Factor binding host plasminogen

### ISSUE THREAD

**Issue by genegodbold** [jhuapl-bio/pathogenesis-gene-ontology#82]
Please enter your suggested changes or comments here.  If your comment relates to a specific term, please provide the PathGO id or rdfs:label of the term in addition to your comment.

For new term requests, please provide the following information:
## Preferred term label
Factor binding host plasminogen

## Synonyms
None

## Textual definition
Parasite protein that binds host plasminogen, a serine proteinase zymogen that circulates in the blood (https://en.wikipedia.org/wiki/Plasmin). This plasminogen recruitment is followed by activation to plasmin and then the bound plasmin proteinase is turned against host immune effectors such as complement.

[There are more than 20 of these for bacteria. Usually the bound host plasmin allows the inactivation of host complement (indirect).]

## Suggested parent term
Along with the newly proposed term "factor binding host fibrinogen", this term should be placed under, uh, maybe PATHGO_0000115 (cellular metabolism disruption factor)?

## Attribution
https://orcid.org/0000-0002-5702-4690

**Comment by jproesch** [comment]
@genegodbold interestingly there is no GO term currently covering binding, but there is one for activation.  What is typically the immediate consequence of the binding for the examples you mention? Activation to plasmin then indirect complement inactivation?

**Comment by genegodbold** [comment]
Yep, the activated plasmin then inactivates complement on the surface of the bacterium.
@jproesch

**Comment by genegodbold** [comment]
Here is an example from Staphylococcus aureus:

Efb is a secreted protein that binds to the complement component C3b and fibrinogen [PMID15243934]. Efb inhibits complement activation and blocks opsonization (and subsequent phagocytosis) in a dose-dependent manner. Efb binds to the alpha-chain of C3 and inhibits both the classical and lectin pathways of complement activation. Efb blocks deposition of C3 on the bacterial surface [PMID15243934]. Efb blocks the formation of the functional C3b opsonin by binding tightly to the thioester-containing domain of the native C3 and by altering its conformation such that it becomes incapable of being processed into C3b [PMID17351618]. Together Sbi and Efb bind complement components C3 and C3b simultaneously with human plasminogen. The bound plasminogen is converted by secreted staphylokinase or by a host-specific urokinase to plasmin which in turn leads to degradation of both C3 and C3b. The bound plasmin also degrades C3a. Sbi and Efb comprise a platform for recruitment of plasminogen, C3 and C3b for efficient degradation of the complement components. Efb enhances the degradation of C3/C3b more than Sbi, probably because it has a higher binding affinity for C3d (Efb dissociation constant for C:Cd3 = 0.3 nanomolar while Sbi = 1.4 micromolar).  Plasmin cleaves C3b at several sites and generates C3c and C3d [PMID23071827].

**Comment by jproesch** [comment]
@genegodbold 

"together Sbi and Efb bind complement components C3 and C3b simultaneously with human plasminogen. The bound plasminogen is converted by secreted staphylokinase or by a host-specific urokinase to plasmin which in turn leads to degradation of both C3 and C3b. The bound plasmin also degrades C3a. Sbi and Efb comprise a platform for recruitment of plasminogen, C3 and C3b for efficient degradation of the complement components."

So in this case plasminogen/plasmin is being recruited to degrade complement?  This fits pretty squarely under "indirect host complement activation"...

**Comment by jproesch** [comment]
@genegodbold I see I had a similar inclination here...to try to fit this in with indirect complement inactivation
